src/components/ebay-button/index.marko

```diff
@@ -40,6 +40,7 @@ $ {
     var fixedHeightClass =
         input.fixedHeight &&
         (sizeClass ? sizeClass + "-fixed-height" : baseClass + "--fixed-height");
+    var variantClass = input.variant ? baseClass + "--" + input.variant : "";
```

Comment:
Here we should do something like:
```
var variant = input.variant || 'standard';
var variantClass = variant !== 'standard' ? baseClass etc
```
Also we should use backticks (Yes it's not in the previous example but that's better to do)
So 
```
`${baseClass}--${variant}`
```

Reply:
Ah, it looks like you already answered that in a comment of your own. Cool.

Reply:
<img width="1279" alt="Screen Shot 2022-02-22 at 1 22 15 PM" src="https://user-images.githubusercontent.com/1755269/155221645-10bd1830-ce1b-4efb-90ff-f23b10054faf.png">

Yes, so just for completeness, this is what happens when it doesn't get added to `ignoredAttribute`

Reply:
Yep, saw that happening...the reason I asked.